frontend/src-admin/app/feedback/feedback-list.component.html

```
@@ -0,0 +1,172 @@
+          <span pTooltip="{{ item.userEmail }}" tooltipPosition="top">
+            {{ item.username }}
+          </span>
+        </td>
+        <td>
+          <span
+            class="message-preview"
+            (click)="openDetail(item)"
+            style="cursor: pointer; text-decoration: underline dotted"
+          >
+            {{ item.message | slice : 0 : 120
+            }}{{ item.message.length > 120 ? "…" : "" }}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)